### ISSUE TITLE
Copy directory timestamp

### ISSUE THREAD

**Issue by Traveler4** [WinMerge/winmerge#1970]
![grafik](https://github.com/WinMerge/winmerge/assets/52359342/6e7cd8ef-1b18-4c0b-b0e0-63678ab62043)

Using Winmerge to harmonise between drives/directories, it is comfortable to use e.g. the command `Copy -> Left to Right`.  

If this operation requires the creation of one or more new directories, then these are created with the current timestamp. This is not ideal when the aim is to have really identical directory structures. Can we get an option which makes a newly created directory have the same timestamp as the original one (or have this as the standard behaviour), please?

You might want to have a look at
- the [robocopy option](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy) `\dcopy:T` to include the directory's timestamp in the copy operation
- [this tool](https://tools.stefankueng.com/SKTimeStamp.html) ([open-source](https://sourceforge.net/p/stefanstools/code/HEAD/tree/trunk/SKTimeStamp/)) to manipulate folder (and file) timestamps

**Comment by luke6464** [comment]
Yes, I also add myself to this request.... it is a fundamental functionality for all tools that perform comparisons and perfect copies of files. It is very strange why this function has not yet been implemented in many tools.
When you copy something from one system to another and you want an "IDENTICAL" copy of everything, the copy must be IDENTICAL in everything including all the creation and modification dates of the files and file/folder names including uppercase and lowercase.
Please add this function in the next versions of WinMerge in addition to the other relating to the comparison between file and folder names with lower/upper case as I already requested in another post ...
[https://github.com/WinMerge/winmerge/issues/2049](url)
Thank you very much for your work.
